mini_CC_BY_SA/metadata.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
All datasets should go to [`croissant/datasets/`](https://github.com/mlcommons/croissant/tree/main/datasets).

Reply:
You can put external data in a separate `data` folder. Example: https://github.com/mlcommons/croissant/tree/main/datasets/coco2014-mini.